packages/skeleton-react/src/lib/components/Segment/Segment.tsx

```diff
@@ -1,113 +1,102 @@
 'use client';
 
-import React, { createContext, useContext, useEffect, useState } from 'react';
+import React, { createContext, useContext, useId } from 'react';
```

Comment:
Lets try to avoid using the `React` default import. So `React.FC` becomes `FC`.

Reply:
I think we've bounced back and forth on this between all components, so we should standardize. I'm fine with that approach.